Index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute page</title>
+</head>
+<body>
+    
```

Comment:
In accordance with best practice, blank lines are discouraged when writing your HTML. Unless, in exceptional cases where it improves readability or helps separate large or logical code blocks.